index.html

```diff
@@ -1470,11 +1549,12 @@ <h3 id="name-token-endpoint-response">
       <h2 id="name-credential-offer">
 <a href="#section-4" class="section-number selfRef">4. </a><a href="#name-credential-offer" class="section-name selfRef">Credential Offer</a>
       </h2>
-<p id="section-4-1">The openid-configuration for an OpenID provider is used to communicate to Clients what capabilities the provider supports, including whether or not it supports the credential issuance flow. Sometime it is desirable to be able to embedded a link to an offer that is invocable by supported Clients.<a href="#section-4-1" class="pilcrow">¶</a></p>
-<p id="section-4-2">The following is a non-normative example of a invocable URL pointing to a credential offer offered by the OpenID Provider <code>issuer.example.com</code><a href="#section-4-2" class="pilcrow">¶</a></p>
-<div class="artwork art-text alignLeft" id="section-4-3">
+<p id="section-4-1"><strong>NOTE this section is still a W.I.P</strong><a href="#section-4-1" class="pilcrow">¶</a></p>
+<p id="section-4-2">The openid-configuration for an OpenID provider is used to communicate to Clients what capabilities the provider supports, including whether or not it supports the credential issuance flow. Sometime it is desirable to be able to embedded a link to an offer that is invocable by supported Clients.<a href="#section-4-2" class="pilcrow">¶</a></p>
+<p id="section-4-3">The following is a non-normative example of a invocable URL pointing to a credential offer offered by the OpenID Provider <code>issuer.example.com</code><a href="#section-4-3" class="pilcrow">¶</a></p>
+<div class="artwork art-text alignLeft" id="section-4-4">
 <pre>openid://offer?https://issuer.example.com/.well-known/openid-configuration#/credential_offers[0]
```

Comment:
is this like a credential manifest?

Reply:
Yeap kind of but a really peeled back version